tests/unit/ssg-module/test_common.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+import ssg.entities.common as common
+
+
+def test_make_items_product_specific():
+    qualified_dict = {
```

Comment:
Why is it called qualified if it doesn't contain a mix of qualified and unqualified items?

Reply:
Renamed.